### DISCUSSION TITLE
How to configure the slug value to use the post/page folder name instead of the title from the frontmatter?

### DISCUSSION THREAD

**Discussion by JeremieLitzler** [CaiJimmy/hugo-theme-stack#868] [Q&A] [answered]
Hi @CaiJimmy 

Your theme appears to use the `title` from the frontmatter to generate the slug if the frontmatter doesn't declare the property `slug`.

I'd like Hugo to use the folder name of the posts and pages instead.

Is that possible?

Thanks.

**Comment by CaiJimmy** [comment, marked as answer]
> Your theme appears to use the title from the frontmatter to generate the slug if the frontmatter doesn't declare the property slug.

That depends on your Hugo configuration, the theme cannot interfere with that aspect. If you are using the starter template, the permalink configuration is stored here: https://github.com/CaiJimmy/hugo-theme-stack-starter/blob/master/config/_default/permalinks.toml

> I'd like Hugo to use the folder name of the posts and pages instead.

I think that's the default behaviour if you remove the permalink configuration. Please give it a try and see if it works.

**Reply by JeremieLitzler** [reply]
I commented the permalink configuration and I get what I was looking for.
